tools/dm/monitor.md

```diff
@@ -12,52 +12,69 @@ category: tools
 
 ## Task
 
+在 Grafana dashboard 中，DM 默认名称为 `DM`。 
```

Comment:
is `DM-task` better?

Reply:
sounds good, then change the dashboard in DM-Ansible?

Reply:
changed